pkg/v3/random/shuffler.go

```diff
@@ -0,0 +1,23 @@
+package random
+
+import "math/rand"
```

Comment:
I see even right now its using math/rand, https://github.com/smartcontractkit/ocr2keepers/blob/main/internal/util/rand.go#L8 not sure why. 

Accepting to unblock for now since its not changing existing behaviour, can create a ticket to investigate this later

Reply:
Behind the scenes the actual reader func is `crand.Read`, but the actual interfaces are `rand.Source` and `rand.Rand`.